resources/quality/ultimaker_methodxl/um_methodxl_2a_um-pva-175_0.2mm.inst.cfg

```diff
@@ -17,60 +17,47 @@ cool_fan_enabled = False
 cool_min_speed = 5
 cool_min_temperature = =material_print_temperature
 initial_layer_line_width_factor = 300
-material_bed_temperature = 45
-material_bed_temperature_layer_0 = 50
 material_final_print_temperature = 210
 material_initial_print_temperature = 210
-material_print_temperature = 210
-material_print_temperature_layer_0 = 220
-material_standby_temperature = 160
 prime_tower_flow = 125
 raft_airgap = 0.0
-raft_base_wall_count = 4
 raft_interface_flow = 110
 raft_interface_infill_overlap = 50
 raft_interface_layers = 1
-raft_interface_line_spacing = 0.9
-raft_interface_line_width = 1
+raft_interface_line_spacing = =1.08 if raft_base_thickness == 0.5 else 0.9
+raft_interface_line_width = =1.2 if raft_base_thickness == 0.5 else 1
 raft_interface_speed = 15
-raft_interface_z_offset = -0.05
+raft_interface_z_offset = =-0.1 if raft_base_thickness == 0.5 else -0.05
```

Comment:
Why the exact comparison with 0.5? Seems like a bit of a weird thing, shouldn't it be someting like lower than 0.5?

Reply:
Paul suggested tying this raft interface logic to a specific raft base value rather than saying "if PVA is paired with N12CF". Although, your comment here made me double check the N12CF file for the raft base height and it was actually set to 0.4 (this is a mistake).  So I am going to update this value in the N12CF file to reflect 0.5 instead of 0.4.  If there's a better way to reference a specific material pair, let me know and I will update it again.  Thank you

Reply:
I tested the formula and it does not switch. Perhaps raft_base_thickness == 0.5 is sensitive to rounding errors
I also tried: =-0.1 if extruderValue(raft_base_extruder_nr, 'material_type') == 'Nylon12 Carbon Fiber' else -0.05
However, that also did not work :-(

Reply:
@wawanbreton just informed me that this is a bug in Cura that they try to resolve before 5.8 beta